package.json

```diff
@@ -25,18 +27,20 @@
     "prepare": "patch-package",
     "prettier-project": "yarn run prettier-write 'src/**/*.{ts,tsx,js,jsx}'",
     "prettier-write": "yarn run prettier --write",
-    "publish-assets:local": "yarn assets && yarn publish-assets",
     "publish-assets": "node scripts/uploadToS3.js",
+    "publish-assets:local": "yarn clean && yarn build:assets && yarn publish-assets",
     "relay": "relay-compiler",
     "sentry": "scripts/sentry.sh",
-    "start:dev": "NODE_ENV=development BUILD_CLIENT=true node --inspect --max_old_space_size=3072 -r dotenv/config ./src",
-    "start:prod:local": "NODE_ENV=production node --inspect --max_old_space_size=3072 -r dotenv/config ./server.dist.js",
-    "start:prod": "yarn assets && yarn build:server && NODE_ENV=production yarn start",
     "start": "scripts/start.sh",
+    "start:dev": "concurrently 'yarn start:dev:relay' 'yarn start:dev:server'",
+    "start:dev:server": "NODE_ENV=development BUILD_CLIENT=true node --inspect --max_old_space_size=3072 -r dotenv/config ./src/index.dev.js",
+    "start:dev:relay": "NODE_ENV=development yarn relay --watch",
```

Comment:
We already have a `yarn relay` command below that the team is familiar with. Do we need this additional script?

Reply:
It's not an additional script, it only exposes the command that was hidden deep in the `start.sh` script. As per the prior conversation removing the overhead of a "side trip" reduces the cognitive overhead.

Reply:
But it's not hidden in start.sh, it's [right here](https://github.com/artsy/force/blob/master/package.json#L30) and folks use it all the time.